packages/tree-widget/src/components/CategoriesTree.scss

```diff
@@ -0,0 +1,69 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@bentley/ui-core/lib/ui-core/index";
+
+.ui-test-app-visibility-widget {
```

Comment:
Do we want to use the ui-test-app scoping here?  For the imodel-select-react scss we prepended all of the names with the package name to try and avoid collisions as much as possible.

Reply:
Definitely a good idea to prepend the rules with the package name because you're right using this scoping will probably end up with collisions. I'll make the changes

Reply:
fixed

Reply:
Adjusted css classes so that they're prepended with the package name. All non-nested classes should have the prefix